src/components/UI/Button/NormalButton/Button.css

```diff
@@ -0,0 +1,16 @@
+.normal-button {
+	background: red;
+	font-weight: 700;
+	font-size: 1.8em;
+	text-align: center;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	padding: 0.3em;
+	/* min-width: 10%; */
+	/* height: 10%; */
+	margin-right: 3em;
+	gap: 0.5em;
+	/* width: 30%; */
```

Comment:
Suggestion : 1. use color variables to handle all the colors at one place.
2. remove commented  code from the project